app/helpers.php

```diff
@@ -78,3 +78,16 @@ function multipleValueQuery($query, $queryString, $filter)
         $query->where($filter, $values[0], 'and');
     }
 }
+
+function getAgeFromBirthdate($birthdate)
+{
+    // Make DateTime for right now
+    $now = new DateTime('now');
+
+    // Make DateTime for birthday
+    $birthday = new DateTime();
+    $birthday->setTimestamp(strtotime($birthdate));
+
+    // Calculate the difference and return the years
+    return $now->diff($birthday)->y;
```

Comment:
This could be simplified a little with Carbon's [difference methods](http://carbon.nesbot.com/docs/#api-difference):

```php
$birthdate = new Carbon($birthdate);
$now = new Carbon();

return $birthdate->diffInYears($now);
```